controllers/multiclusterobservability/multiclusterobservability_controller_test.go

```diff
@@ -242,15 +242,15 @@ func TestMultiClusterMonitoringCRUpdate(t *testing.T) {
 	}
 	//provide a non-existence path to bypass the rendering
 	//cannot convert unstructured.Unstructured into v1.Service in fake client
-	os.Setenv("TEMPLATES_PATH", path.Join(wd, "../../../tests/manifests"))
+	os.Setenv("TEMPLATES_PATH", path.Join(wd, "testdata/manifests"))
```

Comment:
Why we cannot use `../../../tests` as template_path?

Reply:
Looks like we have to test this with directory exists, otherwise we will see lots of warning:
```
2021/06/15 08:36:07 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/observatorium, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/observatorium' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/observatorium' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:07 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/config, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/config' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/config' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:07 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/proxy, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/proxy' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/proxy' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:07 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/grafana, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/grafana' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/grafana' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:07 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:07 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/thanos, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/thanos' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/thanos' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:09 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/observatorium, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/observatorium' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/observatorium' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:09 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/config, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/config' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/config' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:09 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/proxy, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/proxy' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/proxy' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:09 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/grafana, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/grafana' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/grafana' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:09 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:09 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/thanos, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/thanos' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/thanos' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:10 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/observatorium, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/observatorium' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/observatorium' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:10 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/config, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/config' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/config' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:10 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/proxy, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/proxy' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/proxy' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:10 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/grafana, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/grafana' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/grafana' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:10 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:10 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/thanos, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/thanos' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/thanos' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:11 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/observatorium, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/observatorium' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/observatorium' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:11 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/config, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/config' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/config' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:11 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/proxy, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/proxy' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/proxy' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:11 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/grafana, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/grafana' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/grafana' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:11 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/alertmanager' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
2021/06/15 08:36:11 Cannot find this path /go/src/github.com/open-cluster-management/tests/manifests/base/thanos, absolute path error in '/go/src/github.com/open-cluster-management/tests/manifests/base/thanos' : evalsymlink failure on '/go/src/github.com/open-cluster-management/tests/manifests/base/thanos' : lstat /go/src/github.com/open-cluster-management/tests: no such file or directory
```